plover/command/set_config.py

```diff
@@ -0,0 +1,41 @@
+import ast
+
+COMMAND_DELIMITER = ','
+
+
+def plugin_error(e, msg):
+    """ Add an additional message to an exception before raising to the engine. """
+    e.args = (e.args[0] + "\nSET_CONFIG: " + msg,)
+    raise e
+
+
+def set_config(engine, cmdline):
+    """
+    Set a Plover config option to a given value upon executing a stroke pattern.
+
+    Example usage (enter these lines into dictionary):
+    "O*EP": "{PLOVER:SET_CONFIG:translation_frame_opacity,100}",
+    "TR*PB": "{PLOVER:SET_CONFIG:translation_frame_opacity,0}",
+    """
+    # Break down the command string into a key and a value. Throw an error if it isn't in the form of a 2-tuple.
+    try:
+        key_str, value_str = tuple(cmdline.split(COMMAND_DELIMITER, 1))
+    except ValueError as e:
+        plugin_error(e,"Bad command string. Command must be in the form {PLOVER:SET_CONFIG:option,value}")
```

Comment:
I feel like it's unclear that `plugin_error` raises an exception. can we rename to something like `raise_command_error`?